### PR TITLE
clean up views i18n entries

### PR DESCRIPTION
**Pull request by yheuhtozr**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [X] Optimization
- [ ] Documentation Update

## Description

- Delete no longer useful `core` keys that were not able to remove in the scope of each partial PR
- Fix outdated or erroneous i18n messages caused by PRs having crossed each other

## Related Tickets & Documents

#14888

## Added/updated tests?

- [ ] Yes
- [X] No, and this is why: _hopefully does not change any behavior_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
   <div class="crayons-layout crayons-layout--header-inside crayons-layout--2-cols crayons-layout--limited-l">
     <div class="crayons-page-header">
       <% if @tab == "organization" && @organizations.present? %>
-        <h1 class="crayons-title">Settings for
```
Comment (djuber) [inline app/views/users/edit.html.erb:26]: Should this (non-localized) text have been replaced with a translation key?

File: config/locales/views/settings/en.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ en:
             Delete your profile, along with your authentication associations.
 
             This does not include applications permissions. You will have to remove them yourself:
-          desc1b: Delete any and all content you have, such as articles, comments, your reading list or chat messages.
```
Comment (djuber) [inline config/locales/views/settings/en.yml:34]: Thanks for the cleanup here after the connect chat removal!

File: spec/system/homepage/user_visits_homepage_spec.rb [hunk 3]
```diff
@@ -9,9 +9,9 @@
     it "shows the sign-in block" do
       visit "/"
       within "#sidebar-wrapper-left" do
-        p I18n.t("create_account")
-        expect(page).to have_text(I18n.t("core.log_in"))
-        expect(page).to have_text(I18n.t("core.create_account"))
+        p I18n.t("views.main.header.create_account")
+        expect(page).to have_text(I18n.t("views.main.header.login"))
```
Comment (djuber) [inline spec/system/homepage/user_visits_homepage_spec.rb:13]: confirmed these `views.main.header` keys exist, and match the intent, the original core keys are removed above.

**Review by djuber — APPROVED** [review]
minor question about the users/edit view losing a string.

File: app/views/users/edit.html.erb (hunk 1 above)
Comment (yheuhtozr) [reply app/views/users/edit.html.erb:26]: I mean, the equivalent string is added a few lines below:
https://github.com/forem/forem/blob/6ee334f10e25f47c9dc9772291c8b35572bba2b1/app/views/users/edit.html.erb#L31
Currently as I see the line produces a duplicated message. 
image

File: config/locales/views/settings/en.yml (hunk 2 above)
Comment (yheuhtozr) [reply config/locales/views/settings/en.yml:34]: These are reinstatement of #14734 which has overtaken the corresponding i18n PR.

**Review by citizen428 — APPROVED** [review]

**Pull request merged**